src/api/api.js

```diff
@@ -19,3 +19,10 @@ export const fetchComments = (id) => {
     return response.data.comments;
   });
 };
+
+export const voteOnArticle = (id, voteAdjustment) => {
+  return api.patch(`/api/articles/${id}`, {voteAdjustment})
+    .then(response => {
+        return response.data.article
+    });
```

Comment:
Your patch is coming back with a 500 error so isn't currently working. You need to pass in an object with a key of `inc_votes` so it should look like `{inc_votes: 1}`